docs/cluster-configuration.md

```diff
@@ -346,8 +346,8 @@ kind: HyperConverged
 metadata:
   name: kubevirt-hyperconverged
 spec:
-  insecureRegistries:
-    storageImport:
+  storageImport:
```

Comment:
not [StorageImportConfig](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/docs/api.md#storageimportconfig) ?

Reply:
no. `StorageImportConfig` is the name of the **type**. The **field** name is `storageImport`